wsgi.py

```diff
@@ -6,255 +6,348 @@
+            response=json.dumps(
+                {"msg": "project unabled to be created (" + project_uuid + ")"}
+            ),
             status=400,
-            mimetype='application/json'
-            )
+            mimetype="application/json",
+        )
     return Response(
-        response=json.dumps({"msg": "project currently exist (" + project_uuid +")" }),
+        response=json.dumps({"msg": "project currently exist (" + project_uuid + ")"}),
         status=400,
-        mimetype='application/json'
-        )
+        mimetype="application/json",
+    )
+
 
-@application.route("/projects/<project_uuid>", methods=['DELETE'])
-@application.route("/projects/<project_uuid>/owner/<user_name>", methods=['DELETE'])
-def delete_moc_project(project_uuid, user_name=None):
-    (token, openshift_url) = get_token_and_url()
-    if(exists_openshift_project(token, openshift_url, project_uuid)):
-        r = delete_openshift_project( token, openshift_url, project_uuid, user_name)
-        if(r.status_code == 200 or r.status_code == 201):
+@application.route("/projects/<project_uuid>", methods=["DELETE"])
+@auth.login_required
+def delete_moc_project(project_uuid):
+    shift = get_openshift()
+    if shift.project_exists(project_uuid):
+        r = shift.delete_project(project_uuid)
+        if r.status_code == 200 or r.status_code == 201:
             return Response(
-                response=json.dumps({"msg": "project deleted (" + project_uuid +")" }),
+                response=json.dumps({"msg": "project deleted (" + project_uuid + ")"}),
                 status=200,
-                mimetype='application/json'
-               )
+                mimetype="application/json",
+            )
         return Response(
-            response=json.dumps({"msg": "project unabled to be deleted (" + project_uuid +")" }),
+            response=json.dumps(
+                {"msg": "project unabled to be deleted (" + project_uuid + ")"}
+            ),
             status=400,
-            mimetype='application/json'
-            )
+            mimetype="application/json",
+        )
     return Response(
-        response=json.dumps({"msg": "unable to delete, project does not exist(" + project_uuid +")" }),
+        response=json.dumps(
+            {"msg": "unable to delete, project does not exist(" + project_uuid + ")"}
+        ),
         status=400,
-        mimetype='application/json'
-        )
+        mimetype="application/json",
+    )
 
-@application.route("/users/<user_name>", methods=['GET'])
-def get_moc_user(user_name, full_name=None, id_provider="sso_auth", id_user=None):
-    (token, openshift_url) = get_token_and_url()
-    r=None
-    if(exists_openshift_user(token, openshift_url, user_name)):
+
+@application.route("/users/<user_name>", methods=["GET"])
+@auth.login_required
+def get_moc_user(user_name, full_name=None, id_provider="moc-sso", id_user=None):
+    shift = get_openshift()
+    r = None
```

Comment:
ok, I'll run pylint over the codebase.

Reply:
You'll probably want to configure `flake8` to ignore `W503` by putting the following in a `.flake8` file:

```
[flake8]
ignore = W504
```

Otherwise you're going to get hundreds of warnings about a particular indentation style you've used. I don't think the warnings are particular useful, and the volume of warnings masks other problems.

Reply:
Sure, my indentation style is dictated by black's default.